### ISSUE TITLE
Cropping / Imagemanipulation does not work in Frontend

### ISSUE THREAD

**Issue by timwehrmann** [IchHabRecht/filefill#51]
Hey @IchHabRecht 

thanks for this great extension. It helps a lot with getting a local system running without syncing the entire fileadmin, really love it :D

Nevertheless I encountered a problem in two separate instances: Images are cropped with a fixed format using the default TYPO3 image manipulation on the leading TYPO3 instance. The image is cropped and comes out of `_processed_` folder as intended.

On my local instance the original file is taken instead without cropping in frontend. This seems to be nothing but a frontend issue since the cropping mask in `sys_file_reference` is the same on both systems - so from backend perspective everything looks good AFAICS. If I save the cropping and the record / `sys_file_reference` without changing anything, then the image in frontend gets displayed with cropping like on the leading instance and stays like this - so plain saving seems to fix it.

Sadly I cannot hand in a PR with this issue due to not having time yet. Maybe I dig into this later, but before I forget about this I wanted to post it here.

**Comment by IchHabRecht** [comment]
Hi @timwehrmann,

Thank you very much for your issue. Unfortunately I have to tell you, that filefill is designed to not handle any processed files by itself. If the original file is available the file processing can be done locally. As I don't know anything about your project and/or configuration I'm not able to help with this issue.

**Comment by timwehrmann** [comment]
The file gets synced, that is not the problem - dont get me wrong there. The problem is that in both projects only those files that got synced with Filefill are not able to be processed while those that are added manually can be (by both Backend and plain rsync). Interesting part is that it gets fixed by saving the record without having any changes at all. I will dig into this later (today / tomorrow), maybe I'll find out

**Comment by timwehrmann** [comment]
tl;dr: Seems to be a project-wise problem → closed.

---

I set up a plain TYPO3 11 using PHP 7.4 with introduction package.

On "Congratulation" page [id=1] I added a new slide with an background image with a cropping area in the very left bottom corner:

<img width="846" alt="image" src="https://user-images.githubusercontent.com/6337709/154904332-f8df757e-f17c-4797-b850-99147872daf2.png">

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/6337709/154904481-c43cfa3a-c586-4f4b-bd61-1de918552ade.png">

I open now the frontend on remote system and everything works as intended - the cropped image is shown:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/6337709/154904694-d6eeec63-5c1a-4ac8-8d12-f977ace8d8fa.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/6337709/154904721-98f37773-abc9-43c0-9211-6d230a7ff028.png">

Now I synced the database to my local system, the file does not exist at that point there. I then open the frontend and in the very plain TYPO3 11 the file gets displayed cropped just as on remote system 🥳 

In the instances I've spoken of this does not happen, the original file gets displayed without the cropping. So the filefill mechanism works but somehow TYPO3 does not process the file. It does if you do one of two things: Either save the record in which the cropped image is placed or get the file to the system the CLI way using scp and / or stuff. In any way this is not a problem of filefill as far as I can say.
